src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -681,12 +681,12 @@ class ActivityResourceIntTest : IntegrationTestBase() {
               courseScheduleRule()
             }
           }
-          offenderBooking = offender(nomsId = "A1234TT") {
-            booking(agencyLocationId = "LEI") {
+          newOffender(nomsId = "A1234TT") {
```

Comment:
Created a new entry point for Offender that uses the new style of DSL builders etc. This is to allow it to work in conjunction with the existing `offender` entry point. Once all entities related to `offender` have been modelled in the new DSL we can delete it and rename `newOffender`.